_is-int.scss

```diff
@@ -1,14 +1,7 @@
 @function is-int($value) {
     @if type-of($value) != number {
         @warn '`#{$value}` is not a valid number!';
-        @return null;
-    }
-    @else {
-        @if round($value) == $value {
-            @return true;
-        }
-        @else {
-            @return false;
-        }
+        @return false;
```

Comment:
Semantically, you are right, a non-number argument is never an integer. However, this is a breaking change. Uses of the function that check for `null` specifically, differentiating between `null` and `false` for whatever reason would break.
I suggest keeping the `null` return value and merging the simplified code, releasing it as a patch and then creating a new change as a major release to get rid of `null`, which also affects the documentation.

Reply:
No worries. Done.